src/kodeknekkerne/index.md

```diff
@@ -1,22 +1,8 @@
-﻿---
+---
 title: Kodeknekkerne
-external: https://kodeknekkerne.kodegenet.no/
 language: nb
+external: https://kodeknekkerne.kodegenet.no/
+tags:
+    topic: [block_based, step_based]
+    grade: [primary, secondary, junior]
 ---
-
-NRK Skole har utviklet denne serien som er en kombinasjon av gameshow og

```

Comment:
vi kan godt gjøre det slik. da burde vi gjennom alle index.md-filer i kursene og sørge for at første paragraf alltid er kort og konkret, og passer til å beskrive kurset ved hover.